docs/map.json

```diff
@@ -1192,6 +1192,19 @@
             }
           ]
         },
+        {
+          "name": "Database",
```

Comment:
unsure if this should be the heading or not, also should this be plural "Databases"?